backend/python/app/middlewares/validate.py

```diff
@@ -50,10 +51,6 @@ def wrapper(*args, **kwargs):
                 req = json.loads(req_body)
                 req["file"] = request.files.get("file", default=None)
                 dto = dtos[dto_class_name](**req)
-            # } file-storage
-            # no-file-storage {
-            dto = dtos[dto_class_name](**request.json)
```

Comment:
I think this means that requests to the simple-entities endpoint must specify `application/json` as the content type (i.e. a value must be supplied, can't rely on default) in order to follow the correct code path for validation. Could you please add a comment about this in the routes file and the Swagger docs?

Reply:
Axios automatically sets the Content-Type header to `application/json` when the request is a JS object
https://masteringjs.io/tutorials/axios/post-json

Reply:
Got it. Regardless, I think it would be beneficial to document this, in case a different http client is used to make the request :)